src/2d/shallow/topo_module.f90

```diff
@@ -401,8 +404,12 @@ end subroutine set_topo_for_dtopo
 
     subroutine read_topo_file(mx,my,topo_type,fname,topo)
 
+#ifdef NETCDF
+        use netcdf
```

Comment:
It is just checking to see if `-DNETCDF` was set at compile time.  If not it should remove the line. By default the pre-processor flag should be added though, is -cpp really not being used?